modules/core/arrow-data/src/test/kotlin/arrow/data/TryTest.kt

```diff
@@ -330,6 +330,14 @@ class TryTest : UnitSpec() {
       failure.transform({ Try { it.toString() } }) { Try { "NaN" } }.get() shouldBe "NaN"
     }
 
+    "success" {
+      "1".toInt().success() shouldBe success
```

Comment:
There's no need to do a transformation here, as if it fails it'll throw instead of create a `Try` :(

Reply:
Okey! Fixed 😃